src/fontra/client/web-components/range-slider.js

```diff
@@ -223,6 +225,29 @@ export class RangeSlider extends LitElement {
     `;
   }
 
+  handleKeyDown(event) {
+    event.preventDefault();
+    let increment = event.shiftKey ? 10 : 1;
+    let newValue;
+    switch (event.key) {
+      case "ArrowLeft":
+      case "ArrowDown":
+        newValue = this.value - increment;
+        break;
+      case "ArrowRight":
+      case "ArrowUp":
+        newValue = this.value + increment;
+        break;
+    }
+    if (
```

Comment:
It should clamp the value rather than ignore the event. Currently, if the max is 100, and I am at 99, arrow up/right doesn't make it go to 100, likewise if the min is 0 and I am at 1, with arrow down/left.

Reply:
ok